### PR TITLE
Added Header component, tests and storybook. Small style adjustment

### PR DESCRIPTION
**Pull request by roscha80**

### CODE REVIEW

File: src/GlobalStyles.js [hunk 1]
```diff
@@ -10,9 +10,12 @@ const GlobalStyles = createGlobalStyle`
 body {
   font-family: 'Oswald', sans-serif;
   font-size: 112.5%;
-  height: 100vh;
-  max-width: 100%;
+  height: 90vh;
```
Comment (macfrei) [inline src/GlobalStyles.js:13]: Why only `90vh`?

File: src/components/Header.spec.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react'
+import Header from './Header'
+
+describe('Header', () => {
+  it('renders a header with text', () => {
+    const text = 'Headline'
+    render(<Header children={text} />)
+    expect(screen.getByText('Headline')).toBeInTheDocument()
```
Comment (macfrei) [inline src/components/Header.spec.js:8]:
```suggestion
    expect(screen.getByRole('heading', { level: 1 })).toHaveTextContent('Headline')
```

File: src/GlobalStyles.js (hunk 1 above)
Comment (MelGGit) [inline src/GlobalStyles.js:13]: Why did you change it from 100vh to 90 vh?

**Review by MelGGit — APPROVED** [review]
Hey Rob, well done! Everything looks fine for me. I just had a question down below.

File: src/App.js [hunk 3]
```diff
@@ -3,12 +3,14 @@ import CreatePage from './pages/CreatePage'
...
 import { useState } from 'react'
 import useLocalStorage from './hooks/useLocalStorage'
+import Header from './components/Header'
 
 function App() {
   const [users, setUsers] = useLocalStorage('users', [])
   const [currentPage, setCurrentPage] = useState('createPage')
   return (
     <AppWrapper>
+      <Header children={currentPage} />
```
Comment (badabam) [inline src/App.js:13]:
```suggestion
      <Header>{currentPage}</Header>
```

File: src/App.js [hunk 4]
```diff
@@ -33,4 +35,5 @@ export default App
 
 const AppWrapper = styled.div`
```
Comment (badabam) [inline src/App.js:36]: No need to mention `App` again for me in the component name.
```suggestion
const Wrapper = styled.div`
```

File: src/GlobalStyles.js (hunk 1 above)
Comment (badabam) [inline src/GlobalStyles.js:13]: Be careful with this, display sizes vary.

File: src/GlobalStyles.js [hunk 5]
```diff
@@ -10,9 +10,12 @@ const GlobalStyles = createGlobalStyle`
...
+  width: 100%;
+  max-width: 450px;
   overflow-x: hidden !important;
+  display: flex;
+  flex-direction: column;
   align-items: center;
```
Comment (macfrei) [inline src/GlobalStyles.js:19]: Does this actually change anything? Because in the `body` you only have one `div`.

File: src/GlobalStyles.js [hunk 6]
```diff
@@ -10,9 +10,12 @@ const GlobalStyles = createGlobalStyle`
...
   font-family: 'Oswald', sans-serif;
   font-size: 112.5%;
-  height: 100vh;
-  max-width: 100%;
+  height: 90vh;
+  width: 100%;
+  max-width: 450px;
   overflow-x: hidden !important;
+  display: flex;
+  flex-direction: column;
```
Comment (badabam) [inline src/GlobalStyles.js:18]: Please check, if you need this. Block elements stack below each other by default.

**Review by macfrei — COMMENTED** [review]
Nice and small, I like it that! Added a few things! 😺

File: src/components/Header.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+  return <HeadLine>{children}</HeadLine>
+}
+
+const HeadLine = styled.h1`
+  font-size: 1.75rem;
+  background: var(--background_light);
+  color: var(--fontcolor_light);
+  padding: 10px;
+  text-align: center;
+`
```
Comment (badabam) [inline src/components/Header.js:13]:
```suggestion
const Header = styled.h1`
  font-size: 1.75rem;
  background: var(--background_light);
  color: var(--fontcolor_light);
  padding: 10px;
  text-align: center;
`

export default Header
```

File: src/components/Header.spec.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react'
+import Header from './Header'
+
+describe('Header', () => {
+  it('renders a header with text', () => {
+    const text = 'Headline'
+    render(<Header children={text} />)
```
Comment (badabam) [inline src/components/Header.spec.js:7]:
```suggestion
    render(<Header>{text}</Header>)
```

File: src/components/Header.spec.js (hunk 2 above)
Comment (badabam) [inline src/components/Header.spec.js:8]:
```suggestion
    expect(screen.getByRole('heading', {level: 1})).toBeInTheDocument()
```

**Review by badabam — APPROVED** [review]
Well done. Please update your code and 🚢 it!
